src/main/java/com/fundation/logic/view/convert/ConvertVideoPanel.java

```diff
@@ -245,11 +264,28 @@ private void videoCriteriaConvertPanel() {
         gbc_checkBoxThumbnail.gridy = 14;
         add(checkBoxThumbnail, gbc_checkBoxThumbnail);
 
+        JLabel lblThumbnailFormat = new JLabel("Format:");
+        GridBagConstraints gbc_ThumbnailFormat = new GridBagConstraints();
+        gbc_ThumbnailFormat.anchor = GridBagConstraints.EAST;
+        gbc_ThumbnailFormat.insets = new Insets(0, 0, 5, 5);
+        gbc_ThumbnailFormat.gridx = 2;
+        gbc_ThumbnailFormat.gridy = 14;
+        add(lblThumbnailFormat, gbc_ThumbnailFormat);
+
+        comboBoxThumbnailFormat = new JComboBox();
+        comboBoxThumbnailFormat.setModel(new DefaultComboBoxModel(new String[]{"bmp", "png", "jpg"}));
```

Comment:
All previous implemented comboBox show options in uppercase.

Reply:
Solved.